lib/createElement.js

```diff
@@ -0,0 +1,98 @@
+var appendChild = require('./append-child')
```

Comment:
small nit: the other files use dashes rather than camelcase

Reply:
Gotcha